src/en/proposals/fairlysadpanda-revolutionaries.md

```diff
@@ -0,0 +1,333 @@
+
+**As a Rev**: Suddenly plucked from your normal life, you now need to continue your job keeping the station running and overthrow the horrible bosses who’ve been stepping on your neck for years. Work with a secret team, meaning RP opportunities with players you may normally never interact with. Turn your profession into a tool of destruction. Being a regular rev gives normal players a safe opportunity to learn to be an antagonist.
+
+## Why this is fun for the good guys
+
+**As Heads:** You are suddenly the mouse in a station filled with cats, but you have to keep the station running. Can you trust your team? Can you trust anyone? Do you have a weapon close at hand? All the fun kinds of stress. Most department heads suddenly need to become best friends with Sec.
+
+**As Security:** You have multiple departments to defend, and a requirement to temporarily lock up anyone who looks at all shifty. An active competent security team is required to win a Revs round for the crew, and that means the HoS and friends are the stars of the drama.
+
+**As the crew:** Maybe you’ve always wanted to beat your boss senseless. Now you can! But for the strongly-loyal crew, running the station without getting turned is a game of risk-versus-reward. After all, Nanotransen isn’t going to just feed you into a wall of bullets and lasers, but the Head Revs totally might. Pity the medbay; if the round turns into a storming of the palace scenario, they are suddenly going to be very busy.
+
+## Why this is fun for the admins
+
+Revs is a mode where things ramp up in drama over the course of the round, but the regular operation of the station probably isn’t directly at risk. Both sides have specific win conditions (the elimination of the other), with the station being pulled between them. This means it’s different from the normal violence-and-gibbing approach that traitors and operatives cause. This also means no more having multiple layers of anti-metagame legislation to control round flow.
+
+## Important Refactors and New Content
+
+### Head Revolutionary abilities are tied to a Revolutionary Implant.
+
+This implant does three things:
+
+1. Gives the HR innate access to talk on the Syndicate radio.
+2. Grants them a Influence conversion aura around themselves.
+3. Grants them a conversion ability that allows them to covertly flip people to Rev.
+
+Conversion works as follows:
+
+1. The Revolutionary Implant grants the owner the ability to see the Influence level of those around them.
+2. If the Head Revolutionary sees a crew member with a high enough Influence level, they can activate their implant's ability and click on the target.
```

Comment:
Are there any prospective stats or demos for exactly how this will work? Could end up being no different to flashes or too powerful one way or the other so certainly needs prototyping.

Reply:
To-do for tonight is to make a prototype of this